cypress/support/commands.ts

```diff
@@ -227,13 +227,15 @@ export const createTask = (
   orgID?: string,
   name: string = '🦄ask'
 ): Cypress.Chainable<Cypress.Response> => {
-  const flux = `option task = {
-    name: "${name}",
-    every: 24h,
-    offset: 20m
-  }
-  from(bucket: "defbuck")
-        |> range(start: -2m)`
+  const flux = `import "csv"
```

Comment:
Added a generic import here to complicate the task a bit and ensure that another string is declared prior to the task variable